### PR TITLE
New library implementation

### PR DESCRIPTION
**Pull request by ERussel**
- Full Iroha API support implementation add
- Cocoapods support add

### CODE REVIEW

**Comment by neewy** [comment]
AFAIK we need to put license header to all code files. What is a code file and where to find this rule — might be the questions you'll ask but I can't find them now. Maybe somewhere in wiki.hyperledger.org. 

As for now — you can try using grep as Mikhail did here: https://github.com/hyperledger/iroha/pull/1944

**Comment by neewy** [comment]
https://wiki.hyperledger.org/community/copyright-and-license-policy?s[]=license

**Comment by neewy** [comment]
@ERussel any updates?

**Comment by ERussel** [comment]
@neewy, license headers added to all implemented source code, except those generated by protobuf utility. How do think should we also add license header to them?

File: README.md [hunk 1]
```diff
@@ -1,105 +1,46 @@
-# いろはSwift (IrohaSwift)
+# Hyperledger Iroha iOS library
 
-![CocoaPods](https://img.shields.io/cocoapods/v/IrohaSwift.svg)
-![Platform](https://img.shields.io/cocoapods/p/IrohaSwift.svg?style=flat)
+### Please pay attention! Current version of the library was tested and compatible with [`iroha-v1.0.0_rc2`](https://github.com/hyperledger/iroha/releases/tag/1.0.0_rc2).
```
Comment (neewy) [inline README.md]: In general I would use a badge like this: 
`[Iroha 1.0.0-rc2](https://github.com/hyperledger/iroha/releases/tag/1.0.0_rc2)`

File: README.md [hunk 2]
```diff
@@ -1,105 +1,46 @@
...
-```
+7. Consider logs to see if the scenario completed successfully.
 
-### Carthage
-Write in your Cartfile:
-```
-github "hyperledger/iroha-ios.git"
-```
-Run `carthage update`
+Feel free to experiment with example project and don't hesistate to ask any questions.
```
Comment (neewy) [inline README.md]: Where? Can we put these links:
```
* Join [Hyperledger RocketChat](https://chat.hyperledger.org) #iroha channel 
* Use mailing list [hyperledger-iroha@lists.hyperledger.org](mailto:hyperledger-iroha@lists.hyperledger.org)
```

**Comment by neewy** [comment]
@ERussel generated code should not be stored in repo usually. We don't need to put there license headers anyway.

**Comment by ERussel** [comment]
> @ERussel generated code should not be stored in repo usually. We don't need to put there license headers anyway.

I decided that it will be simpler for library user. In this case he/she needs only XCode and just adds library as a dependency to a project. Otherwise we need to require also additional cli tools to be installed.

File: README.md (hunk 2 above)
Comment (ERussel) [reply README.md]: Will make addition PR for this 2 comments.

**Pull request merged**